dist/releases/embeds.go

```diff
@@ -3,25 +3,32 @@ package releases
 import (
 	"embed"
 	"fmt"
+	"os"
 	"path/filepath"
 	"sort"
+	"strings"
+	"syscall"
 
 	semver "github.com/Masterminds/semver/v3"
 )
 
 //go:embed */*
 var configs embed.FS
 
+const JavaFips = "FIPS.java"
+
 // GetVersions returns the list of versions for those embedded configs
 // are available.
 func GetVersions() []string {
 	dirs, err := configs.ReadDir(".")
 	if err != nil { // Should not happen.
 		return nil
 	}
-	names := make([]string, len(dirs))
-	for i, d := range dirs {
-		names[i] = d.Name()
+	names := make([]string, 0, cap(dirs))
```

Comment:
nit: I think it should be `len(dirs)` here (because the cap could be more than len, and we won't add more elements than len).

Reply:
thanks... done